scripts/get-orig-tarball

```diff
@@ -4,20 +4,28 @@ VERBOSITY=0
 TEMP_D=""
 FORMATS=".tar.gz .tar.xz .tar.bz2"
 DSC_BASE_URL="http://launchpad.net/ubuntu/+archive/primary/+files"
+DEBIAN_DSC_BASE_URL="https://deb.debian.org/debian/pool"
```

Comment:
Maybe we can use `https://launchpad.net/debian/` for Debian, and keep the same archive query logic for Ubuntu and Debian, leveraging the fact that LP mirrors Debian? That's what `pull-debian-source` does if I'm not mistaken.

Reply:
+1. I didn't know about that i guess. that is easier. i'll update.